src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -14225,89 +14225,1275 @@ public static void Main()
 5").VerifyDiagnostics();
         }
 
-        [Fact]
-        public void LambdaDefaultParameter_ArrayCommonType_DefaultValueMismatch()
+        private static readonly string s_anonymousDelegateWithDecimalConstant = $$"""
+            .class private auto ansi sealed '<>f__AnonymousDelegate0`1'<T1>
+                extends [{{s_libPrefix}}]System.MulticastDelegate
+            {
+                .custom instance void [{{s_libPrefix}}]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = (
+                	01 00 00 00
+                )
+                // Methods
+                .method public hidebysig specialname rtspecialname 
+                	instance void .ctor (
+                		object 'object',
+                		native int 'method'
+                	) runtime managed 
+                {
+                } // end of method '<>f__AnonymousDelegate0`1'::.ctor
+                .method public hidebysig newslot virtual 
+                	instance void Invoke (
+                		[opt] !T1 arg
+                	) runtime managed 
+                {
+                	.param [1]
+                		.custom instance void [{{s_libPrefix}}]System.Runtime.CompilerServices.DecimalConstantAttribute::.ctor(uint8, uint8, uint32, uint32, uint32) = (
+                			01 00 01 00 00 00 00 00 00 00 00 00 0b 00 00 00
```

Comment:
In particular, what constant this refers to would be useful.

Reply:
Should be apparent from the C# signature in the field's comment now.